src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -159,9 +158,6 @@ const ImageClassificationObservationTable = ({
         const updatedImages = images.filter((f) => f.id !== file.id)
         setImages(updatedImages)
 
-        const updatedUploadedFiles = uploadedFiles.filter((f) => f.id !== file.id)
```

Comment:
we only care about updating images, no need to also keep track of `uploadedFiles`